solutions/standard_solutions/MediaOps/Tutorial_Resource_Studio_Capabilities_And_Capacities.md

```diff
@@ -0,0 +1,110 @@
+
+## Overview
+
+- [Step 1: Create new resource pool](#step-1-create-new-resource-pool)
+- [Step 2: Creating the capability](#step-2-creating-the-capability)
+- [Step 3: Assigning capabilities to resources](#step-3-assigning-capabilities-to-resources)
+- [Step 4: Add resources to the pools](#step-4-add-resources-to-the-pools)
+- [Step 5: Define parameter on the resource pool](#step-5-define-parameter-on-the-resource-pool)
+
+## Step 1: Create new resource pool
+
+1. Open the **Resource Studio** app.
+
+1. Bring up the **Add New Resource Pool** panel by clicking the ``+ New`` button on the upper left corner.
+
+1. Name the new resource "**Converter**".
+
+1. ``Save as Completed`` the new resource pool.
+
+   ![New Converter Resource Pool](~/solutions/images/Resource_Studio_New_Converter_Pool.png)
+
+## Step 2: Creating the capability
+
+1. Navigate to the ``Capability Management`` page.
+
+1. Open the **New Capability** panel by clicking ``+ New Capability`` in the upper left corner.
+
+   ![Capability Management](~/solutions/images/Resource_Studio_Capability_Management.png)
+
+1. Give the new capability ``Name`` "**Conversion Type**".
+
+1. Set the ``Is Mandatory`` toggle on.
+
+1. ``Add Discrete`` options "**IP to SDI**" and "**SDI to IP**".
+
+1. ``Add`` the new capacity.
+
+   ![New Capability](~/solutions/images/Resource_Studio_New_Capability.png)
+
+## Step 3: Assigning capabilities to resources
+
+In this step, we will be adding capabilities to the resources we created in the [Resource Studio Intro Tutorial](xref:Tutorial_MediaOps_Resource_Studio_Intro). If not done yet, It's highly recommended to do that tutorial first.
+
+1. Navigate to the ``Resources`` page.
+
+1. Select the "**Video Solution 001**" resource.
+
+1. On the right side of the screen, you can see 3 sections: ``Capabilities``, ``Capacities``, and ``Properties``. Click on the ``Edit`` button in the ``Capabilities``.
+
+1. In the newly opened ``Manage Capabilities`` window, in the ``CAPABILITIES`` section click on the ``+`` button to add a new capability.
+
+   ![Manage Capabilities](~/solutions/images/Resource_Studio_Manage_Capabilities.png)
+
+1. In the new dropdown, select the ``Conversion Type``.
+
+1. Click on the ``Values...`` button to select the capability values.
+
+1. In the ``DISCRETES`` section, click on the ``+`` button.
+
+1. In the dropdown, select the ``SDI to IP`` option.
+
+1. Click ``Back``.
+
+   ![Manage Capabilities - Discretes](~/solutions/images/Resource_Studio_Manage_Capabilities_Discretes.png)
+
+1. Click ``Apply``.
+
+1. Repeat the steps to add both ``SDI to IP`` and ``IP to SDA`` capability discretes to the "**NewTek 001**" resource.
+ 
+## Step 4: Add resources to the pools
+
+1. Add both **Video Solution 001** and **NewTek 001** resources to the **Converter** resource pool.
+
+## Step 5: Define parameter on the resource pool
+
```

Comment:
I would add a brief explanation here similar as in step 3. That explains why we do this and refers to the tutorial about configuring nodes on jobs.